restapi/restapi.go

```diff
@@ -80,6 +85,12 @@ func (r *RestAPI) ListUsers() ([]personnel_sync.Person, error) {
 		req.SetBasicAuth(r.Username, r.Password)
 	} else if r.AuthType == AuthTypeBearer {
 		req.Header.Set("Authorization", fmt.Sprintf("Bearer: %s", r.Password))
+	} else if r.AuthType == AuthTypeSalesforceOauth {
```

Comment:
[low importance] Now that you have two `else if` statements, it might be prettier to use a case statement.

Reply:
sure, I'll update it